.github/workflows/CI-distro.yml

```
@@ -94,6 +90,10 @@ jobs:
         with:
           version: ${{ matrix.julia-version }}
           arch: ${{ matrix.julia-arch }}
+      - name: 'Install PARI/GP'  # for alnuth, and thus guarana, polenta, radiroot
```

```suggestion
      - name: 'Install PARI/GP'  # for alnuth, and thus guarana, polenta, radiroot
        if: matrix.gap-package == 'alnuth' || matrix.gap-package == 'guarana' || matrix.gap-package == 'polenta' || matrix.gap-package == 'radiroot'
```
we could install it only conditionally like this, but no strong preference